src/main/java/lotto/domain/enums/Prize.java

```diff
@@ -0,0 +1,39 @@
+package lotto.domain.enums;
+
+import java.util.Arrays;
+
+public enum Prize {
+    FIRST(6, 2000000000, Message.FIRST),
```

Comment:
숫자의 자리수가 많아질 때는 `_`로 자리수 구분을 해 줄 수 있습니다.
```suggestion
    FIRST(6, 2_000_000_000, Message.FIRST),
```